tests/integration/docusaurus/expectations/expectation_management.py

```diff
@@ -0,0 +1,56 @@
+# setup
+import great_expectations as gx
+from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.expectations.core import (
+    ExpectColumnValuesToBeInSet,
+    ExpectColumnValuesToNotBeNull,
+)
+
+context = gx.get_context()
+suite_name = "test-suite"
+column_a = "a"
+column_b = "b"
+column_c = "c"
+
+# create a suite
+suite = ExpectationSuite(name=suite_name)
+suite = context.suites.add(suite=suite)
+
+
+# add expectations
+expectation_a = ExpectColumnValuesToBeInSet(
+    column=column_a, value_set=["foo", "bar", "baz"]
+)
+suite.add_expectation(expectation=expectation_a)
+expectation_b = ExpectColumnValuesToNotBeNull(column=column_a)
+suite.add_expectation(expectation=expectation_b)
+
+# find and update a single expectation
+expectation_type = ExpectColumnValuesToNotBeNull
+expectation = next(
+    expectation
+    for expectation in suite.expectations
+    if isinstance(expectation, expectation_type)
+)
+expectation.column = column_b
+expectation.save()
+
+# batch update expectations
```

Comment:
Do we need this example? Feels like you know this out from the previous example. Maybe if we just made 2 comments on the above example like "find a single expectation" and "update expectation"

Reply:
hm, yes i think we do need an example for how to update multiple expectations as a batch, since batch updating expectations works at the suite level, whereas updating a single expectation works at the expectation level. i've renamed to `update multiple expectations as a batch`, lmk if that helps clarify this!

Reply:
Oh I missed that we're saving at the suite level. I'm with you now!